### ISSUE TITLE
prepInputs tests failing due to different extents

### ISSUE THREAD

**Issue by achubaty** [PredictiveEcology/reproducible#93]
Some `prepInputs` tests _not_ run on appveyor/travis/CRAN are failing:

```r
4. Failure: prepInputs doesn't work (@test-prepInputs.R#148) 
5. Failure: prepInputs doesn't work (@test-prepInputs.R#264) 
```

Both fail because of non-matching extents:

```r
> extent(LCC2005)[1:4]
[1] -1473500  -202250  7283000  8563250
> round(extent(StudyAreaCRSLCC2005)[1:4] / 250, 0) * 250
[1] -1473750  -202000  7282750  8563250
```

**Comment by achubaty** [comment]
this is still failing, but now with different values:

```r
> extent(LCC2005)[1:4]
[1] -1710250  -186750  7231750  8678500
> round(extent(StudyAreaCRSLCC2005)[1:4] / 250, 0) * 250
[1] -1473750  -202000  7282750  8563250
```

**Comment by eliotmcintire** [comment]
Fixed with  cc75c4d41efc7078bd317c3c88344db98e196dc3

It was a case of intransitivity of operations...
Needed to `spTransform` the `studyArea` directly in `cropInputs`, instead of the much faster/simpler `spTransform` of the extent of `studyArea`. These are the same if the `studyArea` fills the extent (like a `Raster` does), but it is not correct otherwise.

**Comment by achubaty** [comment]
seems this is back (on macOS at least). need to confirm we didn't lose the prev fix somehow in merge problems. otherwise need to revisit.

**Comment by achubaty** [comment]
this change exists in development branch...so something else is going on, producing similar error.

```
1. Failure: prepInputs doesn't work (part 1) (@test-prepInputs.R#152) 
extent(LCC2005)[1:4] not identical to round(extent(StudyAreaCRSLCC2005)[1:4]/250, 0) * 250.
4/4 mismatches (average diff: 12.6)
[1] -1473725 - -1473750 == 25.067
[2]  -201975 -  -202000 == 25.067
[3]  7282750 -  7282750 ==  0.127
[4]  8563250 -  8563250 ==  0.127
2. Failure: prepInputs doesn't work (part 1) (@test-prepInputs.R#274) 
extent(LCC2005)[1:4] not identical to round(extent(StudyAreaCRSLCC2005)[1:4]/250, 0) * 250.
4/4 mismatches (average diff: 12.6)
[1] -1473725 - -1473750 == 25.067
[2]  -201975 -  -202000 == 25.067
[3]  7282750 -  7282750 ==  0.127
[4]  8563250 -  8563250 ==  0.127
```

**Comment by achubaty** [comment]
resolved in a567b68a835c0af20a0e016bc0436b7d05cdfa54
